src/util/parented_ptr.h

```diff
@@ -104,7 +99,11 @@ class parented_ptr final {
     friend class parented_ptr;
 };
 
+template<typename... Args>
+concept AnyIsQObject = (... || std::is_base_of_v<QObject, std::remove_pointer_t<Args>>);
```

Comment:
welp. lets try again
```suggestion
concept AnyIsQObject = (... || std::is_base_of_v<QObject, std::remove_pointer_t<std::remove_cvref_t<Args>>>);
```